tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/MemberList.kt

```diff
@@ -0,0 +1,80 @@
+package net.corda.cli.plugins.network
+
+import net.corda.cli.api.CordaCliPlugin
+import net.corda.cli.plugins.common.RestClientUtils.createRestClient
+import net.corda.cli.plugins.common.RestCommand
+import net.corda.membership.rest.v1.MemberLookupRestResource
+import net.corda.membership.rest.v1.types.response.RestMemberInfo
+import picocli.CommandLine
+
+@CommandLine.Command(name = "members", description = ["List members"])
+class MemberList : RestCommand(), CordaCliPlugin {
```

Comment:
This could be a `Runnable` similar to the generate group policy command
```suggestion
class MemberList : RestCommand(), Runnable {
```

Reply:
This is what I am getting when I change it to Runnable -> Class 'MemberList' is not abstract and does not implement abstract member public abstract fun run(): Unit defined in java.lang.Runnable

Reply:
Yes you will need to override that `run()`, that will be the entry point for the command so it will need a slight adjustment. You can refer to `GenerateGroupPolicy` which does this